app/helpers/report_helper.rb

```diff
@@ -146,4 +146,129 @@ def reports_group_title
          :group_description => User.current_user.current_group.description}
     end
   end
+
+  def dashboard_widget_summary(widget, sb_data)
+    schedule = widget.miq_schedule
+    timezone = schedule&.run_at ? schedule.run_at[:tz] : session[:user_tz]
+    safe_join([
+                widget_basic_information(widget),
+                widget_status(widget, sb_data, timezone),
+                widget_sb_type_report(widget, sb_data),
+                widget_timer(schedule, timezone, sb_data),
+                widget_visibility(widget),
+              ])
+  end
+
+  private
+
+  def summary_data(title)
+    {:title => _(title), :mode => "miq_widget_#{title.downcase.tr(' ', '_')}"}
+  end
+
+  def row_data(label, value, style = "")
+    data = {:cells => {:label => _(label), :value => value}}
+    data[:style] = style if style.present?
+    data
+  end
+
+  def widget_basic_information(widget)
+    read_only = widget.read_only ? '* ' : ''
+    data = summary_data('Basic Information')
+    data[:rows] = [
+      row_data('Title', "#{read_only}#{widget.title}"),
```

Comment:
Don't all these row_data strings these need to be translated as well?

Reply:
Same here as above, ` 'Title' ` should be translated and then passed in instead of doing ```:label => _(label)```